apps/condo/domains/ticket/components/TicketId/TicketResidentFeatures.tsx

```diff
@@ -1,25 +1,97 @@
+import { BillingReceipt } from '@app/condo/domains/billing/utils/clientSchema'
+import { BankCardIcon } from '@app/condo/domains/common/components/icons/BankCardIcon'
 import { Ticket } from '@app/condo/schema'
 import { MobileIcon } from '@condo/domains/common/components/icons/MobileIcon'
 
 import { Tooltip } from '@condo/domains/common/components/Tooltip'
+import { useIntl } from '@condo/next/intl'
+import { Col, Row } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import dayjs from 'dayjs'
 import get from 'lodash/get'
 import React from 'react'
-import { useIntl } from '@condo/next/intl'
 
-interface TicketResidentFeaturesProps {
-    ticket: Ticket
+interface MobileAppInstalledIndicatorProps {
+    isContactHasMobileApp: boolean
 }
 
-export const TicketResidentFeatures: React.FC<TicketResidentFeaturesProps> = ({ ticket }) => {
+const MobileAppInstalledIndicator: React.FC<MobileAppInstalledIndicatorProps> = ({ isContactHasMobileApp }) => {
     const intl = useIntl()
     const MobileAppInstalledMessage = intl.formatMessage({ id: 'pages.condo.ticket.MobileAppInstalled' })
     const MobileAppNotInstalledMessage = intl.formatMessage({ id: 'pages.condo.ticket.MobileAppNotInstalled' })
 
-    const isContactHasMobileApp = !!get(ticket, 'client')
-
     return (
         <Tooltip title={isContactHasMobileApp ? MobileAppInstalledMessage : MobileAppNotInstalledMessage}>
             <MobileIcon active={isContactHasMobileApp} />
         </Tooltip>
     )
+}
+
+interface PaymentsAvailableIndicatorProps {
+    ticketOrganizationId: string
+}
+
+const getIsPaymentsInMobileAppAvailable = (receiptsInCurrentPeriod, receiptsInPreviousPeriod) => {
+    if (receiptsInCurrentPeriod > 0 || receiptsInPreviousPeriod > 0) {
+        if (receiptsInPreviousPeriod === 0) return true
```

Comment:
this is confusing a bit

Reply:
wrote a comment

Reply:
This comment just repeats what states in the code without getting extra information. Maybe to write something, like "User have new receipts, that are not fetched yet"